p2p/src/net/libp2p/connectivity/tests.rs

```diff
@@ -430,3 +430,26 @@ async fn close_connection() {
     assert_eq!(connmgr.close_connection(&peer_id), Ok(()));
     assert_eq!(connmgr.events.front(), None);
 }
+
+#[tokio::test]
+async fn banned_peer() {
+    let mut connmgr = ConnectionManager::new();
+
+    // known connection
+    let peer_id = PeerId::random();
+
+    connmgr.connections.insert(
```

Comment:
What makes this peer banned in this test? Where is this set?

Reply:
This is just testing how `ConnectionManager` handles banned peers. It's relevant to it what the peer did to get banned.